tests/providerTest.js

```diff
@@ -47,12 +47,7 @@ describe('provider testing', () => {
       opts.publishVerificationResult = true
       opts.pactBrokerToken = process.env.PACTFLOW_TOKEN
       opts.enablePending = true
-      opts.consumerVersionSelectors = [
-        {
-          tag: process.env.DRONE_SOURCE_BRANCH,
-          latest: true
-        }
-      ]
```

Comment:
there is this code (may be this is useful)
```js
export interface ConsumerVersionSelector {
  pacticipant?: string
  tag?: string
  version?: string
  latest?: boolean
  all?: boolean
}
```

Reply:
it seems not to be used (yet)